update_sample_status/main.py

```diff
@@ -106,6 +142,6 @@ def update_sample_status(request):  # pylint: disable=R1710
 
     update_airtable(project_config, sample, status)
 
-    update_sm(project, sample, status, batch)
+    update_sm(project, sample, status, batch, eseqid)
```

Comment:
Is `eseqid` a map of `{name_of_provider: external_id}`, eg: `{"kccg": "EX01"}`?
That's what metamist expects, and what's annotated in `update_sm`, but the variable names make me less confident.

Reply:
Yeah the function kind of assumes that eseqid will be provided as a dict in the request. But I think it's probably better to just take eseqid as a string in the request and create the dict within this script.

Reply:
Will amend, thanks :)